src/main/java/seedu/address/model/person/Address.java

```diff
@@ -7,9 +7,13 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address<MESSAGE_MAX_INPUT_LIMIT> {
```

Comment:
Not sure why there is use of generics here?

Reply:
Typo! Removed it